lib/mayaUsd/resources/scripts/cacheToUsd.py

```diff
@@ -22,19 +22,11 @@
 import re
 
 
-def forceCacheOptions(textOptions):
-    """
```

Comment:
Animation option is no longer forced true. The presence of animated node is used to set it on or off instead, which is done elsewhere.